packages/babel-plugin-transform-destructuring/test/fixtures/destructuring/for-expressions-nest/input.js

```diff
@@ -0,0 +1 @@
+for ( let x in a ) for ( var { } = x ; ; ) ;
```

Comment:
nit: `for-expression-nest` is a bit too general. Maybe `destructuring-empty-in-for-init`?

Reply:
Can I put the two tests together? Or can you suggest another test name?

Reply:
The other can be `destructuring-empty-in-for-body`.